identity/src/main/java/com/android/identity/CredentialData.java

```diff
@@ -261,9 +263,8 @@ static DataItem accessControlProfileToCbor(AccessControlProfile accessControlPro
             X500Name subject = new X500Name("CN=Android Identity Credential Authentication Key");
 
             Date now = new Date();
-            final long kMilliSecsInOneYear = 365L * 24 * 60 * 60 * 1000;
-            Date expirationDate = new Date(now.getTime() + kMilliSecsInOneYear);
-            BigInteger serial = new BigInteger("1");
+            Date expirationDate = new Date(now.getTime() + MILLISECONDS.convert(365, DAYS));
```

Comment:
nit: I personally find `Duration.ofDays(365).toMillis();` more expressive than `MILLISECONDS.convert`

Reply:
Not feasible to do that because java.time.Duration was only added in API 26, but our minSdkVersion is 24.